jump/src/process.rs

```diff
@@ -63,9 +63,20 @@ impl EnvVars {
                     removals.insert(name.to_owned());
                 }
                 EnvVar::RemoveMatching(regex) => {
-                    for (name, _) in env::vars() {
-                        if regex.is_match(name.as_str()) {
-                            removals.insert(name.into());
+                    for (name, _) in env::vars_os() {
+                        match name.into_string() {
+                            Ok(name_utf8) => {
+                                if regex.is_match(&name_utf8) {
+                                    removals.insert(name_utf8.into());
+                                }
+                            }
+                            Err(name_os) => {
+                                warn!(
```

Comment:
This is not an awesome fix. Ideally you'd use a bytes Regex and maybe: https://crates.io/crates/os_str_bytes

Reply:
Filed #148 to address this properly.

Reply:
Fixed in #149.